nullaway/src/main/java/com/uber/nullaway/ErrorBuilder.java

```diff
@@ -302,12 +307,39 @@ boolean symbolHasSuppressWarningsAnnotation(Symbol symbol, String suppression) {
     return false;
   }
 
-  static String errMsgForInitializer(Set<Element> uninitFields) {
+  static int GetLineforField(Element uninitField, VisitorState state) {
+    Tree fieldTree = getTreesInstance(state).getTree(uninitField);
+    DiagnosticPosition position = (DiagnosticPosition) fieldTree;
+    TreePath path = state.getPath();
+    JCCompilationUnit compilation = (JCCompilationUnit) path.getCompilationUnit();
+    JavaFileObject file = compilation.getSourceFile();
+    DiagnosticSource source = new DiagnosticSource(file, null);
+    return source.getLineNumber(position.getStartPosition());
+  }
+
+  static String errMsgForInitializer(Set<Element> uninitFields, VisitorState state) {
     String message = "initializer method does not guarantee @NonNull ";
+    Element uninitField;
     if (uninitFields.size() == 1) {
-      message += "field " + uninitFields.iterator().next().toString() + " is initialized";
+      uninitField = uninitFields.iterator().next();
+      message +=
+          "field "
+              + uninitField.toString()
+              + "(Line:"
```

Comment:
Right now the line number gets formatted as `(Line:XX)`.  I think that `(line XX)` would look better and be more consistent.  @lazaroclapp?

Reply:
I agree with that, I will do this kind of change later.